api/backend/ecs/utilities.go

```diff
@@ -187,10 +188,6 @@ var GetLogs = func(cloudWatchLogs cloudwatchlogs.Provider, taskARNs []*string, t
 
 	logFiles := []*models.LogFile{}
 	for _, logStream := range logStreams {
-		if *logStream.StoredBytes == 0 {
```

Comment:
That's the part i'm working on now. The puzzling thing is I'm unable to reproduce the json marshal error for empty streams using the previous steps. I definitely intend to address it before removing the wip status.